examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
@stubailo  the only reason we use .ng.html is to get something like that, so I guess our HTML source handler will probably be exactly like `static-html`